archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/OutputGenerator.java

```diff
@@ -158,13 +165,42 @@ private List<String> scan(Output.Files files, Context context) {
                          .collect(Collectors.toList());
     }
 
+    private void preRenderTemplate(Path templatePath, MergedModel scope, Context context) {
```

Comment:
Can you experiment to see if you can implement this feature by updating the`builtInModel` method in `MustacheSupport`.